app/mailers/app_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class AppMailer < ActionMailer::Base
-  default from: "info@myflix.com"
+  default from: "hi@timflix.heroku.com"
```

Comment:
The url is actually herokuapp.com, not heroku.com. Doesn't matter much here but that could be crucial in a production app.